Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
Ah, I see it in the [source code](http://opensource.apple.com/source/objc4/objc4-647/runtime/objc-runtime-new.mm) for the Objective-C runtime. Searching for `class_conformsToProtocol` brings it up.